packages/watcher/src/services/EmailSenderService.ts

```diff
@@ -0,0 +1,41 @@
+import sgMail from "@sendgrid/mail";
+import { StayInit } from '../../models/stay';
+
+export default class EmailSenderService {
+  private fromEmail: string;
+  private message: { subject: string; from: string; html: string; to: string };
+  private stayModel: typeof StayInit;
+
+  constructor() {
+    sgMail.setApiKey(process.env.SENDGRID_API_KEY || '');
+    this.fromEmail = process.env.SENDRID_EMAIL_FROM || '';
+  }
+
+  public setMessage(stayModel: typeof StayInit) {
+    this.stayModel = stayModel;
+    const quantity = stayModel.quantity;
+    const startDate = (new Date(stayModel.start_date)).toLocaleDateString("en-US", { timeZone: "UTC" });
+    const end_date = (new Date(stayModel.end_date)).toLocaleDateString("en-US", { timeZone: "UTC" });
+
+    this.message = {
+      to: stayModel.email,
```

Comment:
For testing purposes, it will be good to have a configurable override for `to` email address.
This way, if this parameter will be enabled in the config all emails will be sent to this special address.